scripts/copy-models/index.js

```diff
@@ -23,34 +57,42 @@ const { models } = yargs(process.argv.slice(2))
     .filter((file) => file.isDirectory())
     .map((dir) => join(models.toString(), dir.name, `smithy/model.json`));
 
+  const observedProtocols = new Set();
+
   for (const smithyModelsFile of smithyModelsFiles) {
     try {
       // Test if file exists.
       await fsPromises.stat(smithyModelsFile);
       // File exists, copy it.
+      const absolutePath = resolve(smithyModelsFile);
+
       try {
-        const fileContent = (await fsPromises.readFile(smithyModelsFile)).toString();
+        const model = require(absolutePath);
+        const sdkId = getSdkId(model).toLowerCase().replace(/\s/g, "-");
```

Comment:
confirmed this is the same set of filenames as before and does not create any new files

test: `yarn copy-models -m ...`